printer_mods/README.md

```diff
@@ -210,7 +210,7 @@ like so:
 | | [V0 PiCam Mount](./xbst_/V0_Picam)| A Raspberry Pi Camera Mount For V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | | [DuetZero](./xbst_/DuetZero) | Pi Zero mount over a Duet2 for space saving | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:x:|
 | Xceled | [Single Panel Magnet Mount](./xceled/Single_Panel_Magnet_Mount)| Mount your front panel using only magnets | :x: |:grey_question: |:heavy_check_mark: |:x:|
-| Yeri | [V0 Magnetic Front Panel](./yeri/V0_Magnetic_Front_Panel)| Mount V0 front panel with magnets, removes need for hinges | :heavy_check_mark: | :x: | :x: | :x: |
+| Yeri | [V0 Magnetic Panels](./yeri/V0_Magnetic_Panels)| Mount V0 front panel with magnets, removes need for hinges | :heavy_check_mark: | :x: | :x: | :x: |
```

Comment:
Update the description too, it still says `Mount V0 front panel with magnets`...